frontend/src/pages/FAQ.vue

```diff
@@ -403,9 +403,6 @@
                   Gnosis Chain
                 </a>
               </template>
-              <template v-slot:goerli>
```

Comment:
There is a reference to goerli  on line 364 of this file. Is that still necessary

Reply:
Also it looks like the tests are failing due to a missing api key

Reply:
My guess is because this line was removed https://github.com/ScopeLift/umbra-protocol/pull/541/files#diff-ee35dfb0c99ccd551fb06776ea5a5698ab2ec611e285a0d42ba8ef9a59ffbb4cL13.

Since, mainnet has no break the test is using the optimism api key rather than the ethereum api key.

Reply:
Nice catch, thank you!

Reply:
Fix here https://github.com/ScopeLift/umbra-protocol/pull/541/commits/ca29206aba38e60e19520eb331c13f1754440535